Lazy/ILazy.cs

```diff
@@ -0,0 +1,6 @@
+namespace Lazy;
+
+public interface ILazy<T>
+{
+    public T Get();
```

Comment:
```suggestion
    public ?T Get();
```
Возвращаемые типы в интерфейсе и реализации должны совпадать
(-0.1)